guides/doc-Installing_Proxy/master.adoc

```diff
@@ -26,25 +26,13 @@ include::common/assembly_installing-capsule-server.adoc[leveloffset=+1]
 
 include::common/assembly_performing-additional-configuration-on-smart-proxy-server.adoc[leveloffset=+1]
 
-// Configuring {SmartProxyServer} with External Services
-include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dhcp-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dns-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_using-infoblox-as-dhcp-and-dns-providers.adoc[leveloffset=+1]
-
 :numbered!:
 
 // {SmartProxyServer} Scalability Considerations
 [appendix]
 include::common/modules/ref_smart-proxy-server-scalability-considerations-when-managing-puppet-clients.adoc[leveloffset=+1]
 
 ifndef::satellite[]
-[appendix]
-include::common/modules/ref_dhcp-isc-settings.adoc[leveloffset=+1]
-
 [appendix]
 include::common/modules/ref_dhcp-options-for-network-configuration.adoc[leveloffset=+1]
 endif::[]
```

Comment:
This appendix needs to be dropped too. Then the whole `ifndef` becomes redundant as well.